tests/acceptance/acceptancesuite.js

```diff
@@ -3,17 +3,23 @@ import VerticalPage from './pageobjects/verticalpage';
 import { setupServer, shutdownServer } from './server';
```

Comment:
does running `npx testcafe chrome:headless,firefox:headless tests/acceptance/acceptancesuite.js` work locally?

Reply:
ya, i'm looking into why it failed on circle

Reply:
Re-running it fixed it, which means it's likely an intermittent error out of scope of this PR (the failure occurred in a previous acceptance test)